PR zh/zh_cn readme update

milochen0418 opened the pull request:
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Type of change

Please delete options that are not relevant.

### New Feature Submission:

### Changes To Core Features:

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

picklelo commented:
@milochen0418 I merged in another PR that also updated this. Can you see if there's any additional changes you want to make, may have to rebase on top. Otherwise can close this

milochen0418 commented:
> @milochen0418 I merged in another PR that also updated this. Can you see if there's any additional changes you want to make, may have to rebase on top. Otherwise can close this

@picklelo  I don't know how to rebase on it.  but I adjusted some text for it 
For example,  

安装 Reflex 的同时也会安装 rx 命令行工具.
should change to 
安装 Reflex 的同时也会安装 reflex 命令行工具.  

Because we have the `reflex` command but not `rx` command. 

May I just send another PR to update a little bit for this document?

milochen0418 commented:
It's easier to use another PR to fix some issues in the translation. So I decide to close this PR.
